### ISSUE TITLE
Add class/function for screen/app/window sharing through NetStream

### ISSUE THREAD

**Issue by ROBERT-MCDOWELL** [airsdk/Adobe-Runtime-Support#2376]
## Description

Get access to user screen/app/window to stream it like Javascript in browsers can do with WEBRTC.

- Add a new class/function to start a screenshare exactly (or better, like allow it on mobile devices too) like javascript API Screen Capture and its getDisplayMedia() 
- All users will benefit from this feature

**Comment by henke37** [comment]
The easiest api design would be to have a new class representing a capture session and then adding an `NetStream.attachCaptureSessionThingy` method.

**Comment by ROBERT-MCDOWELL** [comment]
yes it can be a good start. it would be very nice to use the javascript engine to get the canvas  from which it needs to stream live and send it to a worker, like we can do in browser today.
